FirebaseMLModelDownloader/Sources/ModelInfo.swift

```diff
@@ -52,20 +58,19 @@ struct ModelInfo {
       let path = defaults.value(forKey: "\(defaultsPrefix).model-path") as? String else {
       return nil
     }
-    self.name = name
+    name = modelName
     self.downloadURL = url
     self.modelHash = modelHash
     self.size = size
     self.path = path
   }
 
-  func writeToDefaults(app: FirebaseApp, defaults: UserDefaults) throws {
+  func save(toDefaults defaults: UserDefaults, appName: String) throws {
```

Comment:
I feel like the old name was better `writeToDefaults`. Either that, or `saveToDefaults(_:appName:)` works as well based on your preference.

Reply:
Gotcha, updated to `writeToDefaults(_:appName:)`.